package.json

```diff
@@ -42,7 +43,8 @@
     "dev": "next -p 8080",
     "startup": "./startup.bat",
     "build": "next build",
-    "start": "npm rebuild && node ./server/server.js",
+    "start-dev": "node ./server/server.js",
+    "start": "node ./server/server.js",
```

Comment:
shouldnt this be - 
"start-dev": "npm rebuild && node ./server/server.js",
    "start": "npm rebuild && node ./server/server.js",

Reply:
do we need to have rebuild on start-dev .. in that case we can do that

Reply:
we keep this as is, unless you see the dev change needs. i will raise a new PR with those changes